docs/configuring-playbook-own-webserver-caddy.md

```diff
@@ -0,0 +1,90 @@
+
```

Comment:
This `docs/configuring-playbook-own-webserver-caddy.md` file is pretty much the same as `examples/reverse-proxies/caddy2-in-container/README.md`.

It's better to remove this one and leave the other.